pages/management/configuration.md

```diff
@@ -0,0 +1,81 @@
+---
+title: Configuration Variables
+---
+
+# Configuration Variables
+
+Configuration variables allow you to provide runtime configuration to the host OS and supervisor. These variables all begin with `RESIN_`. Beginning with supervisor v7.0.0, a number of them appear automatically in your dashboard when your device is provisioned.
+
+Configuration variables can be managed at the fleet and device levels.
```

Comment:
I think this would read better as 'Configuration variables can be managed at both the fleet and device level'